app/styles/global.css

```diff
@@ -423,45 +424,40 @@ body {
 }
 
 @media screen and (min-width: 500px) and (max-width: 782px) {
-  .font-size-6-25 {
-    font-size: 5.5rem;
-  }
-  .width-37-5 {
-    width: 17rem;
+  .width-35 {
```

Comment:
i understand that you are changing the class properties based on screen size

seems a bit confusing that now width-35 applies 32rem

why are you doing this this way? are you following docs or an article or something?